tests/Benchmarks/Benchmarks.csproj

```diff
@@ -9,6 +9,11 @@
     <Copyright>Microsoft Corporation, All rights reserved</Copyright>
     <NoWarn>$(NoWarn);CS0618</NoWarn>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="BenchmarkDotNet.Artifacts\**" />
```

Comment:
What is the purpose of this ItemGroup?

Reply:
by default, VS displays the content of BenchmarkDotNet.Artifacts folder in the solution explorer. The folder contains results and logs. This changes is the result of right clicking on the folder and choosing "exclude from project"